quantity.hpp

```diff
@@ -124,75 +124,76 @@ class implicit_quantity_caster : protected quantity<U,S,V,T> {
  *
  * This specialization assures quantities are created with tags.
  *
- * @tparam       Unit   the quantity's unit type
+ * @tparam  Dimension   the quantity's dimension type
```

Comment:
for this `@tparam` list to be correct you would have to list all numerator / denumerator parameters.